.github/workflows/CI.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  build_zip:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ghidra: [ 'latest', '10.1.3', '10.1.2', '10.1.1' ]
+    steps:
+      - name: Set Ghidra version
+        id: ghidra_version
+        run: |
+          if [ ${{ matrix.ghidra }} = latest ]; then
+            ghidra_version=$(curl -fLsSo /dev/null -w %{url_effective} https://github.com/NationalSecurityAgency/ghidra/releases/latest |
```

Comment:
Hmm seems rather fragile? Maybe use `https://api.github.com/repos/NationalSecurityAgency/ghidra/releases/latest` and `jq` to get the name from JSON. I'm fine with specifying the values manually too.